Update monaco peer dependency

I tried running `npm install` and got the following error message. It's probably because I'm using the new npm v7.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: monaco-editor@0.22.3
npm ERR! node_modules/monaco-editor
npm ERR!   monaco-editor@"^0.22.3" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer monaco-editor@"^0.21.2" from @monaco-editor/loader@1.0.0
npm ERR! node_modules/@monaco-editor/loader
npm ERR!   @monaco-editor/loader@"^1.0.0" from the root project
```

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
     "build": "rollup -c && cp ./src/types.d.ts ./lib/"
   },
   "peerDependencies": {
-    "monaco-editor": "^0.21.2"
+    "monaco-editor": ">= 0.21.0 < 0.24.0"
   },
   "keywords": [
     "monaco",

